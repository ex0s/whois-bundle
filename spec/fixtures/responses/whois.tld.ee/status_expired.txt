%  This Whois Server contains information on 
% Estonian Top Level Domain ee TLD

domain:       tenerife.ee
registrant:   CID:EEDIRECT:410154
admin-c:      CID:EEDIRECT:410154
nsset:        NSSID:EEDIRECT:405838
registrar:    eedirect
status:       Expired
registered:   04.07.2010 05:05:30
expire:       05.01.2011

contact:      CID:EEDIRECT:410154
org:          ARUMÄE MAJA OÜ
name:         Marko Põlluäär
e-mail:       Not Disclosed - Visit www.eestiinternet.ee for webbased WHOIS
registrar:    eedirect
created:      04.07.2010 05:05:30

nsset:        NSSID:EEDIRECT:405838
nserver:      ns1550.hostgator.com (174.132.145.195)
nserver:      ns1549.hostgator.com (174.132.145.194)
tech-c:       CID:EEDIRECT:410154
registrar:    eedirect
created:      04.07.2010 05:05:30


