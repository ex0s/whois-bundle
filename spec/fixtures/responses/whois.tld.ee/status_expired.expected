#status
  %s == :expired

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{04.07.2010 05:05:30}

#updated_on
  %s == nil

#expires_on
  %s %CLASS{time}
  %s %TIME{05.01.2011}


#nameservers
  %s %CLASS{array}
  %s %SIZE{2}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns1550.hostgator.com"
  %s[0].ipv4 == "174.132.145.195"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns1549.hostgator.com"
  %s[1].ipv4 == "174.132.145.194"
